__tests__/app.test.js

```diff
@@ -19,3 +19,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an object containing an array of topics, each having 'slug' and 'description'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body}) => {
+        console.log({body});
+        expect(body.topics).toHaveProperty("GET /api/topics");
+        body.topics["GET /api/topics"].exampleResponse.topics.forEach(topic => {
```

Comment:
I am not entirely sure what's going on here, but generally, you should only need to destructure topics from body, check whether it's an array and whether it has the length we expect. Even better to check that it has slug + description, but we definitely want to check that we get the amount of topics back that we expect!